client/src/app/user/shared/register.effects.spec.ts

```diff
@@ -0,0 +1,25 @@
+import { TestBed, inject } from '@angular/core/testing';
+import { provideMockActions } from '@ngrx/effects/testing';
+import { Observable } from 'rxjs';
+
+import { RegisterEffects } from './register.effects';
+
+describe('RegisterEffects', () => {
+  let actions$: Observable<any>;
```

Comment:
__[TSLint]__ Identifier 'actions$' is never reassigned; use 'const' instead of 'let'. (prefer-const)

<sub>posted by Sider</sub>